### PR TITLE
WebPublisher: Updated Settings

### PR DESCRIPTION
**Pull request by kalisp**
## Brief description
Changed settings to list instead of dictionary to allow single frame and multiframe sequences into same family

## Description
Former implementation was dictionary, so single family couldn't be assigned to sequences (pngs) or single file (mp4).

Screenshot 2022-02-03 171456

## Testing notes:
1. Update `project_settings/webpublisher/publish/CollectPublishedFiles/task_type_to_family`
2. Publish single file or sequence

### CODE REVIEW

**Comment by mkolar** [comment]
Task linked: [OP-2579 Webpublisher: redesign the settings](https://app.clickup.com/t/6658547/OP-2579)

**Comment by iLLiCiTiT** [comment]
Why do we have subset name template there? Shouldn't the subset name be created using `get_subset_name`?

**Comment by kalisp** [comment]
Used `get_subset_name_with_asset_doc` function instead of field in Settings.

File: openpype/hosts/webpublisher/webserver_service/webpublish_routes.py [hunk 1]
```diff
@@ -359,12 +359,19 @@ async def get(self, project_name=None) -> Response:
             "studio_exts": set(["psd", "psb", "tvpp", "tvp"])
         }
         collect_conf = sett["webpublisher"]["publish"]["CollectPublishedFiles"]
-        for _, mapping in collect_conf.get("task_type_to_family", {}).items():
-            for _family, config in mapping.items():
-                if config["is_sequence"]:
-                    configured["sequence_exts"].update(config["extensions"])
-                else:
-                    configured["file_exts"].update(config["extensions"])
+        configs = collect_conf.get("task_type_to_family", [])
```
Comment (iLLiCiTiT) [inline openpype/hosts/webpublisher/webserver_service/webpublish_routes.py:362]: Few questions:
1. Why are you loading settings manyally and not using `self.task_type_to_family`?
2. Why is the task type in settings if is not used?
3. Shouldn't you store the family from key of `conf_mappings` if `conf_mappings` is dictionary?

Comment (iLLiCiTiT) [reply openpype/hosts/webpublisher/webserver_service/webpublish_routes.py:362]: Nevermind. I though this is in the collector but this is differnt file. Damn...

Comment (kalisp) [reply openpype/hosts/webpublisher/webserver_service/webpublish_routes.py:362]: Yes, this is only for frontend to collect which formats are "allowed" eg. configured. Currently only per project_name.

**Review by iLLiCiTiT — APPROVED** [review]
Code looks ok but would be better if @64qam would test it.

**Review by 64qam — APPROVED** [review]

**Pull request merged**